src/main/java/core/basesyntax/operation/ReturnOperation.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.operation;
+
+import core.basesyntax.db.FruitStorage;
+import core.basesyntax.model.FruitTransaction;
+
+public class ReturnOperation implements OperationHandler {

```

Comment:
The class `ReturnOperation` should not begin with an empty line. Please remove the redundant empty line at the start of the class definition.